tables/AlternateMans/AntennaPairFile.h

```diff
@@ -199,11 +199,17 @@ class AntennaPairFile {
   void ReadHeader() {
     unsigned char data[kHeaderSize];
     file_.ReadHeader(data);
+    if (!std::equal(data, data + 8, kMagicHeaderTag)) {
+      throw std::runtime_error(
+          "The Antenna-pair columnar file header not have the expected tag for "
```

Comment:
I have a feeling there's a "does" missing in this sentence, shouldn't it be something like: "... file header does not have ..."?

Reply:
Yes indeed, good catch :)

Reply:
(now fixed)